### PR TITLE
Add FHIR documentation

### PR DESCRIPTION
**Pull request by madushadhanushka**
Fix https://github.com/wso2/docs-ei/issues/1711

### CODE REVIEW

File: en/micro-integrator/docs/references/connectors/fhir-connector/fhir-connector-example.md [hunk 1]
```diff
@@ -16,6 +16,35 @@ To know the further information about these operations please refer this link.
...
+```
+[[custom_message_builders]]
+content_type = "application/fhir+json"
+class = "org.wso2.micro.integrator.core.json.JsonStreamBuilder"
+[[custom_message_formatters]]
+content_type = "application/fhir+json"
+class = "org.wso2.micro.integrator.core.json.JsonStreamFormatter"
+```
+
+If you are using **EI 6**, you can enable this property by doing the following Axis2 configurations in the **<PRODUCT_HOME>\repository\conf\axis2\axis2.xml** file.
```
Comment (Nirothipan) [inline en/micro-integrator/docs/references/connectors/fhir-connector/fhir-connector-example.md:34]: we are giving both windows and linux paths. shall we just give only linux path.

Also do we need to mention about EI 6, in this doc 
@nilminiwso2

File: en/micro-integrator/docs/references/connectors/fhir-connector/fhir-connector-example.md [hunk 2]
```diff
@@ -16,6 +16,35 @@ To know the further information about these operations please refer this link.
 
 > **Note**: If no ID element is provided, or the value is wrong, the server responds with a HTTP 400 error code and provides an operation outcome identifying the issue.
 
+Before you start configuring the FHIR connector, you also need WSO2 MI, and we refer to that location as `<PRODUCT_HOME>`.
+
+Specific message builders/formatters configuration needs to be enabled in the product as shown below before starting the WSO2 EI service.
```
Comment (Nirothipan) [inline en/micro-integrator/docs/references/connectors/fhir-connector/fhir-connector-example.md:21]: EI Service ?

File: en/micro-integrator/docs/references/connectors/fhir-connector/fhir-connector-example.md [hunk 3]
```diff
@@ -16,6 +16,35 @@ To know the further information about these operations please refer this link.
...
+
+If you are using **EI7**, you need to enable this property by adding the following to the **<PRODUCT_HOME>/conf/deployment.toml** file. You can further refer to the [Working with Message Builders and Formatters](../../../references/config-catalog/#http-transport) and [Product Configurations](../../micro-integrator/setup/message_builders_formatters/message-builders-and-formatters/) documentations.
```
Comment (Nirothipan) [inline en/micro-integrator/docs/references/connectors/fhir-connector/fhir-connector-example.md:23]: enable which property ?

File: en/micro-integrator/docs/references/connectors/fhir-connector/fhir-connector-example.md (hunk 1 above)
Comment (samgnaniah) [reply en/micro-integrator/docs/references/connectors/fhir-connector/fhir-connector-example.md:34]: Yes, we need to. For connector docs, our approach is to provide docs for the connector version, regardless of which version of EI is used. So even though this resides in the 7.x space, it is still relevant.

File: en/micro-integrator/docs/references/connectors/fhir-connector/fhir-connector-example.md [hunk 4]
```diff
@@ -16,6 +16,35 @@ To know the further information about these operations please refer this link.
 
 > **Note**: If no ID element is provided, or the value is wrong, the server responds with a HTTP 400 error code and provides an operation outcome identifying the issue.
 
+Before you start configuring the FHIR connector, you also need WSO2 MI, and we refer to that location as `<PRODUCT_HOME>`.
+
+Specific message builders/formatters configuration needs to be enabled in the product as shown below before starting the WSO2 MI server.
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/fhir-connector/fhir-connector-example.md:21]:
```suggestion
Before starting the product, you need to enable configurations for specific message builders/formatters.
```

**Review by Nirothipan — APPROVED** [review]

**Review by samgnaniah — APPROVED** [review]

**Pull request merged**